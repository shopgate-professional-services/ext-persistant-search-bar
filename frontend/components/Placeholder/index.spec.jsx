import React from 'react';
import { mount } from 'enzyme';
import { Placeholder } from './index';

describe('<Placeholder />', () => {
  it('should render', () => {
    const component = mount(<Placeholder />);
    expect(component).toMatchSnapshot();
  });
});
