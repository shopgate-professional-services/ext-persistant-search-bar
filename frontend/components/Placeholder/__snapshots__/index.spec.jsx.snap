// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Placeholder /> should render 1`] = `
<Placeholder>
  <div
    className="css-g9nz84"
  />
</Placeholder>
`;
